### PR TITLE
Unit tests related to SDL fixes

### PR DESCRIPTION
**Pull request by dkalinowski**
CVS-129566

### CODE REVIEW

File: src/test/pythonnode_test.cpp [hunk 1]
```diff
@@ -1187,6 +1187,24 @@ TEST_F(PythonFlowTest, PythonCalculatorTestSingleInTwoOutTwoParallelExecutorsWit
...
 
 TEST_F(PythonFlowTest, ConverterWithMissingTagMap) {
+    ConstructorEnabledModelManager manager{"", getPythonBackend()};
+    std::string testPbtxt = R"(
+    input_stream: "OVTENSOR:in"
+    output_stream: "OVMS_PY_TENSOR:out"
+        node {
+            name: "pythonNode1"
+            calculator: "PyTensorOvTensorConverterCalculator"
+            input_stream: "OVTENSOR:in"
```
Comment (dkalinowski) [inline src/test/pythonnode_test.cpp:1197]: TODO: Add test with:
```
[type.googleapis.com / mediapipe.PyTensorOvTensorConverterCalculatorOptions]: {

                }
```

**Review by atobiszei — APPROVED** [review]

File: src/test/streaming_test.cpp [hunk 2]
```diff
@@ -1301,6 +1301,35 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+TEST_F(StreamingTest, FirstRequestRestrictedParamName) {
```
Comment (mzegla) [inline src/test/streaming_test.cpp:1304]: what happens if the second request comes with restricted param name?

File: src/test/pythonnode_test.cpp (hunk 1 above)
Comment (mzegla) [reply src/test/pythonnode_test.cpp:1197]: Could you also add such test for PythonExecutorCalculator?

Comment (atobiszei) [reply src/test/pythonnode_test.cpp:1197]: If this is specific mediapipe behavior then IMO it doens't make sense to multiply such tests. Maybe add single test to mediapipe_framework that will show that such formating of options is not allowed?

File: src/test/streaming_test.cpp (hunk 2 above)
Comment (dkalinowski) [reply src/test/streaming_test.cpp:1304]: Second request parameters are not deserialized. Only the first request can deliver those.

**Review by mzegla — APPROVED** [review]

**Pull request merged**